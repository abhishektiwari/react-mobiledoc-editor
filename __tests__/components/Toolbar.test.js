import React from 'react';
import { shallow } from 'enzyme';
import Toolbar from '../../src/components/Toolbar';

describe(Toolbar, () => {
  it('passes props to first child', () => {
    const wrapper = shallow(<Toolbar className="foo" />);
    expect(wrapper).toMatchSnapshot();
  });
});
