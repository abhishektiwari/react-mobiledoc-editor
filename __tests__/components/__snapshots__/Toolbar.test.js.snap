// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar passes props to first child 1`] = `
<ul
  className="foo"
>
  <li>
    <MarkupButton
      tag="strong"
    />
  </li>
  <li>
    <MarkupButton
      tag="em"
    />
  </li>
  <li>
    <LinkButton />
  </li>
  <li>
    Style: 
    <SectionSelect
      tags={
        Array [
          "h1",
          "h2",
          "h3",
        ]
      }
    />
  </li>
  <li>
    <SectionButton
      tag="blockquote"
    />
  </li>
  <li>
    <SectionButton
      tag="ul"
    >
      UL
    </SectionButton>
  </li>
  <li>
    <SectionButton
      tag="ol"
    >
      OL
    </SectionButton>
  </li>
</ul>
`;
